disparate-flies/WorkforceManagement
# added Description column into Training Program

Please structure your pull requests this way:

- **Purpose of PR**
Discovered that we needed a Description column in the TrainingPrograms table. Went into the SQL file and added in the Description column and descriptions for each Training Program. 

- **Issue number this request impacts** ------
#22 

- **Description of what was changed:**
Discovered that we needed a Description column in the TrainingPrograms table. Went into the SQL file and added in the Description column and descriptions for each Training Program. 

- **Thorough list of steps to test**

1. From Master, fetch my branch ay-updated-SQL using the 'git fetch --all' command
2. Check into my branch
3. Type the 'ls' command to see all the files
4. Type 'start SQL-HR-IT.sql'
5. When the program opens, highlight and uncomment the top portion of the code, Alter Table, Delete, Drop
6. While the top part of the code is still highlighted, hit the execute button. (This should clear all the data in the tables and then drop the tables so that you can start building the database from fresh)
7. Then uncomment the top part of the code out again and then execute the rest of the code (This should create your tables and the data)
8. As long as you don't get any errors, everything should be working correctly



if you don't do this you will be
**FIRED**
